src/tests/math/test_math.f90

```diff
@@ -0,0 +1,132 @@
+! SPDX-Identifier: MIT
```

Comment:
This file `test_math.f90` should not be present in the PR. Instead it should be generated when invoking cmake or make.
Please remove it from the PR.

Reply:
Okay I will do that.

Reply:
Yes, only one or the other should be committed. As I understand it, test_math.fypp is not used.

Reply:
Yes!, `.fypp` file is not used. I will remove `.fypp` file from PR within 2 days.